bundler/spec/quality_spec.rb

```diff
@@ -144,6 +144,7 @@ def check_for_specific_pronouns(filename)
       gem.coc
       gem.linter
       gem.mit
+      gem.bundle
```

Comment:
I see no need to skip all these settings, particularly since we already document some of them. But it's fine to skip it for now like already done for other similar settings 👍.